# PredictionMSG
# Zhiang Chen, Aug 2016
string name1
float32 value1
string name2
float32 value2
